content/docs/secure/encryption/rekey.md

```diff
@@ -31,143 +32,109 @@ image: dellemc/csm-encryption-rekey-controller:v0.1.0
 #  Never: Never pull the image.
 imagePullPolicy: IfNotPresent
 
+# logLevel: Log level of the rekey controller.
+# Allowed values: "error", "warning", "info", "debug", "trace".
+logLevel: "info"
+
 # This value is required and must match encryption.pluginName value
 # of the corresponding Dell CSI driver.
-provisioner: sec-isilon.dellemc.com
-# This represents the port number used by encryption driver Rest API server.
-port: 3838
+provisioner:
+
+# This value is required and must match encryption.apiPort value
+# of the corresponding Dell CSI driver.
+port:
 ```
-> **NOTE**: It is important that the `provisioner` reflect the same name as the `pluginName` specified in the standard encryption values.yaml file, and the `port`
-reflect the same port number as the encryption driver REST API server would be listening on. If they don't match then all rekey attempts will fail.
+
+| Parameter | Description                                                                                               | Required | Default |
+| --------- |-----------------------------------------------------------------------------------------------------------|----------| ------- |  
+| image | Rekey controller image name.                                                                              | No | Yes |
+| imagePullPolicy | Rekey controller image pull policy.                                                                       | No | Yes |
+| logLevel | Log level of the rekey controller.                                                                        | No | Yes |
+| provisioner | This value is required and must match `encryption.pluginName` value of the corresponding Dell CSI driver. | Yes | No |
+| port | This value is required and must match `encryption.apiPort` value of the corresponding Dell CSI driver.    | Yes | No |

```

Comment:
The Default column should contain the actual default values, if present. E.g. "info" for logLevel, leave empty for port.

Reply:
lol

Reply:
fixed.